README
NAME

    Dist::Zilla::PluginBundle::Author::BBYRD - DZIL Author Bundle for BBYRD

SYNOPSIS

        ; Very similar to...
     
        [ReportPhase]
     
        ; Makefile.PL maker
        [MakeMaker]
     
        [Authority]
        authority = cpan:BBYRD
        locate_comment = 1
     
        [Git::NextVersion]
        first_version = 0.90
     
        [Git::GatherDir]
     
        ; File modifiers
        [OurPkgVersion]
        [PodWeaver]
        config_plugin = @Author::BBYRD
     
        ; File pruners
        [PruneCruft]
     
        ; Extra file creation
        [GitFmtChanges]
        log_format = [%h]%n* Author: %an <%ae>%n* Date:   %ai (%ar)%n%n%B
     
        [ManifestSkip]
        [Manifest]
        [License]
        [ReadmeAnyFromPod / ReadmePodInRoot]    ; Pod README for Root (for GitHub, etc.)
        [ReadmeAnyFromPod / ReadmeTextInBuild]  ; Text README for Build
        [ReadmeAnyFromPod / ReadmeHTMLInBuild]  ; HTML README for Build (never POD, so it doesn't get installed)
        [InstallGuide]
        [ExecDir]
     
        ; Many tests
        [@TestingMania]
        disable = Test::Perl::Critic
        disable = Test::EOL
        disable = Test::Kwalitee
        disable = Test::Pod::LinkCheck
        disable = MetaTests
        changelog = CHANGES
     
        ; POD tests
        ;[Test::PodSpelling]  ; Win32 install problems
     
        ; Other xt/* tests
        [RunExtraTests]
        ;[MetaTests]  ; until Test::CPAN::Meta supports 2.0
        [Test::EOL]
        trailing_whitespace = 0
     
        [Test::CheckDeps]
        ;[Test::Pod::LinkCheck]  ; Both of these are borked...
        ;[Test::Pod::No404s]     ; ...I really need to create my own
        [Test::ReportPrereqs]
        [Test::CheckManifest]
     
        ; Prereqs
        [@Prereqs]
        minimum_perl = 5.10.1
     
        [CheckPrereqsIndexed]
     
        ; META maintenance
        [MetaConfig]
        [MetaJSON]
        [MetaYAML]
     
        [MetaNoIndex]
        directory = t
        directory = xt
        directory = examples
        directory = corpus
     
        [MetaProvides::Package]
        meta_noindex = 1        ; respect prior no_index directives
     
        [GithubMeta]
        issues = 1
        user   = SineSwiper
     
        [ContributorsFromGit]
     
        [MetaResources]  ; only loaded if needed
        x_IRC = $x_IRC
     
        ; Post-build plugins
        [CopyFilesFromBuild]
        move = .gitignore
        move = README.pod
     
        ; Post-build Git plugins
        [TravisYML]
        notify_email = 0
        notify_irc = irc://irc.perl.org/#sanity
        ; used for Travis::TestRelease
        support_builddir = 1
        ; keep sanity from balking at these
        post_before_install_build = cpanm --quiet --notest --skip-satisfied autovivification indirect multidimensional
     
        [Git::CheckFor::CorrectBranch]
        [Git::CommitBuild]
        branch =
        release_branch = build/%b
        release_message = Release build of v%v (on %b)
     
        [@Git]
        allow_dirty = dist.ini
        allow_dirty = .travis.yml
        allow_dirty = README.pod
        changelog =
        commit_msg = Release v%v
        push_to = origin master:master
        push_to = origin build/master:build/master
     
        [GitHub::Update]
        metacpan = 1
     
        [TestRelease]
        [Travis::TestRelease]
        create_builddir = 1
     
        [ConfirmRelease]
        [UploadToCPAN]
        [InstallRelease]
        [Clean]
     
        ; sanity deps
        ; authordep autovivification
        ; authordep indirect
        ; authordep multidimensional

DESCRIPTION

    I frelling hate these things, but several releases in, I found myself
    needing to keep my dist.ini stuff in sync, which requires a single
    module to bind them to.

NAMING SCHEME

    I'm a strong believer in structured order in the chaos that is the CPAN
    namespace. There's enough cruft in CPAN, with all of the forked
    modules, legacy stuff that should have been removed 10 years ago, and
    confusion over which modules are available vs. which ones actually
    work. (Which all stem from the same base problem, so I'm almost
    repeating myself...)

    Like I said, I hate writing these personalized modules on CPAN. I even
    bantered around the idea of using MetaCPAN's author JSON input
    <https://github.com/SineSwiper/Dist-Zilla-PluginBundle-BeLike-You/blob/
    master/BeLike-You.pod> to store the plugin data. However, keeping the
    Author plugins separated from the real PluginBundles is a step in the
    right direction. See KENTNL's comments on the Author namespace for more
    information.

CAVEATS

    This uses Dist::Zilla::Role::PluginBundle::Merged, so all of the
    plugins' arguments are available, using Merged's rules. Special care
    should be made with arguments that might not be unique with other
    plugins. (Eventually, I'll throw these into config_rename.)

    If this is a problem, you might want to consider using @Filter.

    One exception is x_IRC, which is detected and passed to MetaResources
    properly.

SEE ALSO

    In building my ultimate dist.ini file, I did a bunch of research on
    which modules to cram in here. As a result, this is a pretty large set
    of plugins, but that's exactly how I like my DZIL. Feel free to
    research the modules listed here, as there's a bunch of good modules
    that you might want to include in your own dist.ini and/or Author
    bundle.

AVAILABILITY

    The project homepage is
    https://github.com/SineSwiper/Dist-Zilla-PluginBundle-Author-BBYRD.

    The latest version of this module is available from the Comprehensive
    Perl Archive Network (CPAN). Visit http://www.perl.com/CPAN/ to find a
    CPAN site near you, or see
    https://metacpan.org/module/Dist::Zilla::PluginBundle::Author::BBYRD/.

SUPPORT

 Internet Relay Chat

    You can get live help by using IRC ( Internet Relay Chat ). If you
    don't know what IRC is, please read this excellent guide:
    http://en.wikipedia.org/wiki/Internet_Relay_Chat. Please be courteous
    and patient when talking to us, as we might be busy or sleeping! You
    can join those networks/channels and get help:

      * irc.perl.org

      You can connect to the server at 'irc.perl.org' and talk to this
      person for help: SineSwiper.

 Bugs / Feature Requests

    Please report any bugs or feature requests via
    https://github.com/SineSwiper/Dist-Zilla-PluginBundle-Author-BBYRD/issu
    es.

AUTHOR

    Brendan Byrd <BBYRD@CPAN.org>

CONTRIBUTOR

    Sergey Romanov <complefor@rambler.ru>

COPYRIGHT AND LICENSE

    This software is Copyright (c) 2014 by Brendan Byrd.

    This is free software, licensed under:

      The Artistic License 2.0 (GPL Compatible)



CHANGES
Revision History for Dist-Zilla-PluginBundle-Author-BBYRD
=========================================================

v1.04 2014-10-19
----------------

[a16c76d]
* Author: Brendan Byrd <Perl@ResonatorSoft.org>
* Date:   2014-10-02 00:38:38 -0400 (2 weeks ago)

Add support for x_IRC


v1.03 2014-08-02
----------------

[2a2b5bc]
* Author: Brendan Byrd <Perl@ResonatorSoft.org>
* Date:   2014-08-02 00:47:58 -0400 (3 months ago)

Release v1.03

[458722e]
* Author: Brendan Byrd <Perl@ResonatorSoft.org>
* Date:   2014-08-02 00:44:48 -0400 (3 months ago)

Add PluginBundle requirements


v1.02 2014-05-03
----------------

[d9635da]
* Author: Brendan Byrd <Perl@ResonatorSoft.org>
* Date:   2014-05-03 19:50:55 -0400 (6 months ago)

Release v1.02

[447eaf1]
* Author: Brendan Byrd <Perl@ResonatorSoft.org>
* Date:   2014-05-03 01:17:36 -0400 (6 months ago)

Add open_status_url option to Travis::TestRelease

[8cc1bc8]
* Author: Brendan Byrd <Perl@ResonatorSoft.org>
* Date:   2014-05-02 22:20:44 -0400 (6 months ago)

Use ::Role::BundleDeps

[39fbc80]
* Author: Brendan Byrd <Perl@ResonatorSoft.org>
* Date:   2014-05-02 22:19:21 -0400 (6 months ago)

Add authority

[cb2fcb0]
* Author: Brendan Byrd <Perl@ResonatorSoft.org>
* Date:   2014-05-02 21:00:02 -0400 (6 months ago)

Add Travis::TestRelease and other Travis changes

Add @TestingMania


v1.01 2014-05-01
----------------

[05039c0]
* Author: Brendan Byrd <Perl@ResonatorSoft.org>
* Date:   2014-05-01 22:40:38 -0400 (6 months ago)

Release v1.01

[236fdfc]
* Author: Brendan Byrd <Perl@ResonatorSoft.org>
* Date:   2014-05-01 21:54:59 -0400 (6 months ago)

Change NoTabsTests to Test::NoTabs

Also, fix some TravisYML settings

[f8b6e7e]
* Author: Brendan Byrd <Perl@ResonatorSoft.org>
* Date:   2014-01-29 22:17:07 -0500 (9 months ago)

Add in Prereqs for module changes from v1.00


v1.00 2013-11-26
----------------

[7a40e7d]
* Author: Brendan Byrd <Perl@ResonatorSoft.org>
* Date:   2013-11-26 00:04:49 -0500 (11 months ago)

Release v1.00

[dfe62b3]
* Author: Brendan Byrd <Perl@ResonatorSoft.org>
* Date:   2013-11-23 23:16:43 -0500 (11 months ago)

Merge branch 'master' of github.com:SineSwiper/Dist-Zilla-PluginBundle-Author-BBYRD

[7f9dba6]
* Author: Brendan Byrd <Perl@ResonatorSoft.org>
* Date:   2013-11-23 23:09:58 -0500 (11 months ago)

Change around a few modules

+Test::CPAN::Changes
-MetaResourcesFromGit
+GithubMeta

Add formatting to GitFmtChanges

[101a295]
* Author: Sergey Romanov <complefor@rambler.ru>
* Date:   2013-11-16 06:31:52 +0400 (11 months ago)

change PWP::Encoding to PWP::SingleEncoding


===================================
End of changes in the last 365 days
===================================


profiles/default/Module.pm
package {{$name}};

# VERSION
# ABSTRACT: ---insert abstract here---

#############################################################################
# Modules

use sanity;
use Moo;
use MooX::Types::MooseLike::Base qw();

### INSERT MODULES HERE ###

use namespace::clean;
no warnings 'uninitialized';

#############################################################################
# Attributes

### INSERT ATTRS HERE ###

#############################################################################
# Pre/post-BUILD

around BUILDARGS => sub {
   my ($orig, $self) = (shift, shift);
   my $hash = shift;
   $hash = { $hash, @_ } unless ref $hash;

   ### INSERT CODE HERE ###

   $orig->($self, $hash);
};

#############################################################################
# Methods

### INSERT CODE HERE ###

42;

__END__

=begin wikidoc

= SYNOPSIS

   # code

= DESCRIPTION

### Ruler ########################################################################################################################12345

Insert description here...

= CAVEATS

### Ruler ########################################################################################################################12345

Bad stuff...

= SEE ALSO

### Ruler ########################################################################################################################12345

Other modules...

= ACKNOWLEDGEMENTS

### Ruler ########################################################################################################################12345

Thanks and stuff...

=end wikidoc


lib/Pod/Weaver/PluginBundle/Author/BBYRD.pm
package Pod::Weaver::PluginBundle::Author::BBYRD;

our $AUTHORITY = 'cpan:BBYRD'; # AUTHORITY
our $VERSION = '1.04'; # VERSION
# ABSTRACT: Pod::Weaver Author Bundle for BBYRD

use sanity;

use Pod::Weaver 3.101635; # fixed ABSTRACT scanning
use Pod::Weaver::Config::Assembler;

# Dependencies
use Pod::Weaver::Plugin::WikiDoc ();
use Pod::Weaver::Section::Availability ();
use Pod::Elemental::Transformer::List 0.101620 ();
use Pod::Weaver::Section::Support 1.001        ();
use Pod::Weaver::Section::Contributors ();

sub _exp { Pod::Weaver::Config::Assembler->expand_package( $_[0] ) }

### TODO: Consider a simplified version of this a la DZIL:R:PB:Merged?
sub mvp_bundle_config {
   my @plugins;
   push @plugins, (
      # [-SingleEncoding]
      # [-WikiDoc]
      # [@CorePrep]
      #
      # [Name]

      [ '@Author::BBYRD/SingleEncoding', _exp('-SingleEncoding'), {} ],
      [ '@Author::BBYRD/WikiDoc',        _exp('-WikiDoc'),        {} ],
      [ '@Author::BBYRD/CorePrep',       _exp('@CorePrep'),       {} ],
      [ '@Author::BBYRD/Name',           _exp('Name'),            {} ],

      # [Region / prelude]
      #
      # [Generic / SYNOPSIS]
      # [Generic / DESCRIPTION]
      # [Generic / OVERVIEW]

      [ '@Author::BBYRD/Prelude',     _exp('Region'),  { region_name => 'prelude' } ],
      [ '@Author::BBYRD/Synopsis',    _exp('Generic'), { header      => 'SYNOPSIS' } ],
      [ '@Author::BBYRD/Description', _exp('Generic'), { header      => 'DESCRIPTION' } ],
      [ '@Author::BBYRD/Overview',    _exp('Generic'), { header      => 'OVERVIEW' } ],
   );

   foreach my $plugin (
      # [Collect / ATTRIBUTES]
      # command = attr
      #
      # [Collect / METHODS]
      # command = method
      #
      # [Collect / FUNCTIONS]
      # command = func

      [ 'Attributes',   _exp('Collect'), { command => 'attr' } ],
      [ 'Methods',      _exp('Collect'), { command => 'method' } ],
      [ 'Functions',    _exp('Collect'), { command => 'func' } ],
     )
   {
       $plugin->[2]{header} = uc $plugin->[0];
       push @plugins, $plugin;
   }

   push @plugins, (
      # [Leftovers]
      #
      # [Region / postlude]
      #
      # [Availability]

      [ '@Author::BBYRD/Leftovers',    _exp('Leftovers'),    {} ],
      [ '@Author::BBYRD/postlude',     _exp('Region'),       { region_name => 'postlude' } ],
      [ '@Author::BBYRD/Availability', _exp('Availability'), {} ],

      # [Support]
      # perldoc = 0
      # websites = none
      # repository_link = none
      # bugs = metadata
      # bugs_content = Please report any bugs or feature requests via {WEB}.
      # irc = irc.perl.org, #distzilla, SineSwiper

      [ '@Author::BBYRD/Support', _exp('Support'), {
         perldoc            => 0,
         websites           => 'none',
         repository_link    => 'none',
         bugs               => 'metadata',
         bugs_content       => 'Please report any bugs or feature requests via {WEB}.',
         ### XXX: Use a DZIL stash to store the IRC channel? ###
         irc                => 'irc.perl.org, SineSwiper',
      } ],

      # [Authors]
      # [Contributors]
      # [Legal]
      #
      # [-Transformer]
      # transformer = List

      [ '@Author::BBYRD/Authors',      _exp('Authors'),      {} ],
      [ '@Author::BBYRD/Contributors', _exp('Contributors'), {} ],
      [ '@Author::BBYRD/Legal',        _exp('Legal'),        {} ],
      [ '@Author::BBYRD/List',         _exp('-Transformer'), { 'transformer' => 'List' } ],
   );

   return @plugins;
}

42;

__END__

=pod

=encoding UTF-8

=head1 NAME

Pod::Weaver::PluginBundle::Author::BBYRD - Pod::Weaver Author Bundle for BBYRD

=head1 SYNOPSIS

    ; Very similar to...
 
    [-SingleEncoding]
    [-WikiDoc]
    [@CorePrep]
 
    [Name]
 
    [Region / prelude]
 
    [Generic / SYNOPSIS]
    [Generic / DESCRIPTION]
    [Generic / OVERVIEW]
 
    [Collect / ATTRIBUTES]
    command = attr
 
    [Collect / METHODS]
    command = method
 
    [Collect / FUNCTIONS]
    command = func
 
    [Leftovers]
 
    [Region / postlude]
 
    [Availability]
    [Support]
    perldoc = 0
    websites = none
    repository_link = none
    bugs = metadata
    bugs_content = Please report any bugs or feature requests via {WEB}.
    irc = irc.perl.org, SineSwiper
 
    [Authors]
    [Contributors]
    [Legal]
 
    [-Transformer]
    transformer = List
 
    ; PodWeaver deps
    ; authordep Pod::Weaver::Plugin::WikiDoc
    ; authordep Pod::Weaver::Section::Availability
    ; authordep Pod::Weaver::Section::Support
    ; authordep Pod::Elemental::Transformer::List

=head1 DESCRIPTION

Like the DZIL one, this is a personalized Author bundle for my Pod::Weaver
configuration.

=head1 NAMING SCHEME

I'm a strong believer in structured order in the chaos that is the CPAN
namespace.  There's enough cruft in CPAN, with all of the forked modules,
legacy stuff that should have been removed 10 years ago, and confusion over
which modules are available vs. which ones actually work.  (Which all stem
from the same base problem, so I'm almost repeating myself...)

Like I said, I hate writing these personalized modules on CPAN.  I even bantered
around the idea of using L<MetaCPAN's author JSON input|https://github.com/SineSwiper/Dist-Zilla-PluginBundle-BeLike-You/blob/master/BeLike-You.pod>
to store the plugin data.  However, keeping the Author plugins separated from the
real PluginBundles is a step in the right direction.  See
L<KENTNL's comments on the Author namespace|Dist::Zilla::PluginBundle::Author::KENTNL/NAMING-SCHEME>
for more information.

=head1 AVAILABILITY

The project homepage is L<https://github.com/SineSwiper/Dist-Zilla-PluginBundle-Author-BBYRD>.

The latest version of this module is available from the Comprehensive Perl
Archive Network (CPAN). Visit L<http://www.perl.com/CPAN/> to find a CPAN
site near you, or see L<https://metacpan.org/module/Dist::Zilla::PluginBundle::Author::BBYRD/>.

=head1 AUTHOR

Brendan Byrd <BBYRD@CPAN.org>

=head1 COPYRIGHT AND LICENSE

This software is Copyright (c) 2014 by Brendan Byrd.

This is free software, licensed under:

  The Artistic License 2.0 (GPL Compatible)

=cut


Makefile.PL

# This file was automatically generated by Dist::Zilla::Plugin::MakeMaker v5.020.
use strict;
use warnings;

use 5.010001;

use ExtUtils::MakeMaker ;

use File::ShareDir::Install;
$File::ShareDir::Install::INCLUDE_DOTFILES = 1;
$File::ShareDir::Install::INCLUDE_DOTDIRS = 1;
install_share module => "Dist::Zilla::MintingProfile::Author::BBYRD", "profiles";


my %WriteMakefileArgs = (
  "ABSTRACT" => "DZIL Author Bundle for BBYRD",
  "AUTHOR" => "Brendan Byrd <BBYRD\@CPAN.org>",
  "CONFIGURE_REQUIRES" => {
    "File::ShareDir::Install" => "0.06"
  },
  "DISTNAME" => "Dist-Zilla-PluginBundle-Author-BBYRD",
  "EXE_FILES" => [],
  "LICENSE" => "artistic_2",
  "NAME" => "Dist::Zilla::PluginBundle::Author::BBYRD",
  "PREREQ_PM" => {
    "Dist::Zilla" => "5.020",
    "Dist::Zilla::Plugin::Authority" => "1.006",
    "Dist::Zilla::Plugin::CheckPrereqsIndexed" => "0.012",
    "Dist::Zilla::Plugin::Clean" => "0.07",
    "Dist::Zilla::Plugin::ContributorsFromGit" => "0.014",
    "Dist::Zilla::Plugin::CopyFilesFromBuild" => "0.103510",
    "Dist::Zilla::Plugin::Git" => "2.025",
    "Dist::Zilla::Plugin::Git::CheckFor::CorrectBranch" => "0.011",
    "Dist::Zilla::Plugin::GitFmtChanges" => "0.005",
    "Dist::Zilla::Plugin::GitHub::Update" => "0.38",
    "Dist::Zilla::Plugin::GithubMeta" => "0.46",
    "Dist::Zilla::Plugin::InstallGuide" => "1.200003",
    "Dist::Zilla::Plugin::InstallRelease" => "0.008",
    "Dist::Zilla::Plugin::MetaProvides::Package" => "2.001002",
    "Dist::Zilla::Plugin::MojibakeTests" => "0.7",
    "Dist::Zilla::Plugin::OurPkgVersion" => "0.005001",
    "Dist::Zilla::Plugin::PodWeaver" => "4.006",
    "Dist::Zilla::Plugin::ReadmeAnyFromPod" => "0.142470",
    "Dist::Zilla::Plugin::ReportPhase" => 0,
    "Dist::Zilla::Plugin::RunExtraTests" => "0.022",
    "Dist::Zilla::Plugin::Test::CPAN::Changes" => "0.008",
    "Dist::Zilla::Plugin::Test::CPAN::Meta::JSON" => "0.003",
    "Dist::Zilla::Plugin::Test::CheckDeps" => "0.012",
    "Dist::Zilla::Plugin::Test::CheckManifest" => "0.04",
    "Dist::Zilla::Plugin::Test::Compile" => "2.051",
    "Dist::Zilla::Plugin::Test::DistManifest" => "2.000004",
    "Dist::Zilla::Plugin::Test::EOL" => "0.12",
    "Dist::Zilla::Plugin::Test::MinimumVersion" => "2.000005",
    "Dist::Zilla::Plugin::Test::NoTabs" => "0.09",
    "Dist::Zilla::Plugin::Test::Portability" => "2.000005",
    "Dist::Zilla::Plugin::Test::ReportPrereqs" => "0.019",
    "Dist::Zilla::Plugin::Test::UnusedVars" => "2.000006",
    "Dist::Zilla::Plugin::Travis::TestRelease" => "1.12",
    "Dist::Zilla::Plugin::TravisYML" => "1.12",
    "Dist::Zilla::PluginBundle::Git" => "2.025",
    "Dist::Zilla::PluginBundle::Prereqs" => "0.93",
    "Dist::Zilla::PluginBundle::TestingMania" => "0.23",
    "Dist::Zilla::Role::BundleDeps" => "0.002001",
    "Dist::Zilla::Role::PluginBundle::Merged" => "1.00",
    "Dist::Zilla::Role::PluginBundle::PluginRemover" => "0.102",
    "Moose" => "2.1213",
    "Pod::Coverage::TrustPod" => "0.100003",
    "Pod::Elemental::Transformer::List" => "0.101620",
    "Pod::Weaver" => "4.006",
    "Pod::Weaver::Plugin::WikiDoc" => "0.093003",
    "Pod::Weaver::Section::Availability" => "1.20",
    "Pod::Weaver::Section::Contributors" => "0.008",
    "Pod::Weaver::Section::Support" => "1.001",
    "Test::CPAN::Meta::JSON" => "0.15",
    "sanity" => "1.03"
  },
  "TEST_REQUIRES" => {
    "Test::CheckDeps" => "0.010",
    "Test::More" => "0.94"
  },
  "VERSION" => "1.04",
  "test" => {
    "TESTS" => "t/*.t"
  }
);


my %FallbackPrereqs = (
  "Dist::Zilla" => "5.020",
  "Dist::Zilla::Plugin::Authority" => "1.006",
  "Dist::Zilla::Plugin::CheckPrereqsIndexed" => "0.012",
  "Dist::Zilla::Plugin::Clean" => "0.07",
  "Dist::Zilla::Plugin::ContributorsFromGit" => "0.014",
  "Dist::Zilla::Plugin::CopyFilesFromBuild" => "0.103510",
  "Dist::Zilla::Plugin::Git" => "2.025",
  "Dist::Zilla::Plugin::Git::CheckFor::CorrectBranch" => "0.011",
  "Dist::Zilla::Plugin::GitFmtChanges" => "0.005",
  "Dist::Zilla::Plugin::GitHub::Update" => "0.38",
  "Dist::Zilla::Plugin::GithubMeta" => "0.46",
  "Dist::Zilla::Plugin::InstallGuide" => "1.200003",
  "Dist::Zilla::Plugin::InstallRelease" => "0.008",
  "Dist::Zilla::Plugin::MetaProvides::Package" => "2.001002",
  "Dist::Zilla::Plugin::MojibakeTests" => "0.7",
  "Dist::Zilla::Plugin::OurPkgVersion" => "0.005001",
  "Dist::Zilla::Plugin::PodWeaver" => "4.006",
  "Dist::Zilla::Plugin::ReadmeAnyFromPod" => "0.142470",
  "Dist::Zilla::Plugin::ReportPhase" => 0,
  "Dist::Zilla::Plugin::RunExtraTests" => "0.022",
  "Dist::Zilla::Plugin::Test::CPAN::Changes" => "0.008",
  "Dist::Zilla::Plugin::Test::CPAN::Meta::JSON" => "0.003",
  "Dist::Zilla::Plugin::Test::CheckDeps" => "0.012",
  "Dist::Zilla::Plugin::Test::CheckManifest" => "0.04",
  "Dist::Zilla::Plugin::Test::Compile" => "2.051",
  "Dist::Zilla::Plugin::Test::DistManifest" => "2.000004",
  "Dist::Zilla::Plugin::Test::EOL" => "0.12",
  "Dist::Zilla::Plugin::Test::MinimumVersion" => "2.000005",
  "Dist::Zilla::Plugin::Test::NoTabs" => "0.09",
  "Dist::Zilla::Plugin::Test::Portability" => "2.000005",
  "Dist::Zilla::Plugin::Test::ReportPrereqs" => "0.019",
  "Dist::Zilla::Plugin::Test::UnusedVars" => "2.000006",
  "Dist::Zilla::Plugin::Travis::TestRelease" => "1.12",
  "Dist::Zilla::Plugin::TravisYML" => "1.12",
  "Dist::Zilla::PluginBundle::Git" => "2.025",
  "Dist::Zilla::PluginBundle::Prereqs" => "0.93",
  "Dist::Zilla::PluginBundle::TestingMania" => "0.23",
  "Dist::Zilla::Role::BundleDeps" => "0.002001",
  "Dist::Zilla::Role::PluginBundle::Merged" => "1.00",
  "Dist::Zilla::Role::PluginBundle::PluginRemover" => "0.102",
  "Moose" => "2.1213",
  "Pod::Coverage::TrustPod" => "0.100003",
  "Pod::Elemental::Transformer::List" => "0.101620",
  "Pod::Weaver" => "4.006",
  "Pod::Weaver::Plugin::WikiDoc" => "0.093003",
  "Pod::Weaver::Section::Availability" => "1.20",
  "Pod::Weaver::Section::Contributors" => "0.008",
  "Pod::Weaver::Section::Support" => "1.001",
  "Test::CPAN::Meta::JSON" => "0.15",
  "Test::CheckDeps" => "0.010",
  "Test::More" => "0.94",
  "sanity" => "1.03"
);


unless ( eval { ExtUtils::MakeMaker->VERSION(6.63_03) } ) {
  delete $WriteMakefileArgs{TEST_REQUIRES};
  delete $WriteMakefileArgs{BUILD_REQUIRES};
  $WriteMakefileArgs{PREREQ_PM} = \%FallbackPrereqs;
}

delete $WriteMakefileArgs{CONFIGURE_REQUIRES}
  unless eval { ExtUtils::MakeMaker->VERSION(6.52) };

WriteMakefile(%WriteMakefileArgs);

{
package
MY;
use File::ShareDir::Install qw(postamble);
}


